src/utils.zig

```diff
@@ -376,6 +417,10 @@ pub fn FixedSizeCircularBuffer(T: type, capacity: comptime_int) type { // MARK:
 			}
 		}
 
+		pub fn dequeueFront(self: *Self) ?T {
+			return self.dequeue();
```

Comment:
There is a mismatch in the naming conventions.
Below, in `CircularBufferQueue`, you delcared `dequeue_back` as `self.dequeue`

Reply:
Yes, but I suppose we prefer `camelCase` rigth? That one should be gone + it was done that way because of other methods being `snake_case`

Reply:
I have removed changes from `CircularBufferQueue`

Reply:
It's also about back vs front naming

Reply:
Hmm, right, I'll swap those